lib/line/bot/v2/messaging_api/api/messaging_api_blob_client.rb

```diff
@@ -47,12 +47,13 @@ def initialize(base_url: nil, channel_access_token:, http_options: {})
 
           # Download image, video, and audio data sent from users.
           # This requests to <code>GET https://api-data.line.me/v2/bot/message/{messageId}/content</code>
+          # This returns an array containing response, HTTP status code, and header in order. Please specify all header keys in lowercase.
           #
           # @param message_id [String] Message ID of video or audio
           # @see https://developers.line.biz/en/reference/messaging-api/#get-content
-          # @return [response body, response status code, and response headers]
           # @return [Array((String|nil), Integer, Hash{String => String})] when HTTP status code is 200
-          def get_message_content_with_http_info(
+          # @return [Array((String|nil), Integer, Hash{String => String})] when other HTTP status code is returned. String is HTTP response body itself.
+          def get_message_content_with_http_info( # steep:ignore MethodBodyTypeMismatch 
```

Comment:
by ignoring MethodBotyTypeMismatch, we can't detect inconsistency between code and RBS. As PR descrition, I checked there is no mistake (as of now), it's steep's internal mistake. (ttps://github.com/soutaro/steep/issues/16030)